conans/util/log.py

```diff
@@ -36,6 +36,19 @@ def configure_logger(logging_level=logging.CRITICAL, logging_file=None):
     return logger
 
 
+def get_log_level_by_name(level_name):
```

Comment:
I would move this to the `conf.__init__` module as it is not used anywhere else.